lib/ansible/utils/module_docs_fragments/nso.py

```diff
@@ -34,4 +34,6 @@ class ModuleDocFragment(object):
   password:
     description: NSO password
     required: true
+requirements:
+  - Cisco NSO version 4.4.3 or higher
```

Comment:
Is this true for *all* NSO modules that ship with Ansible?

Reply:
At least for now. As long as they convert schema information to the JSON-RPC path format. If it changes the requirements could be moved back into the separate modules?

Reply:
Correct, sounds good.